data/items.js

```diff
@@ -2599,7 +2602,9 @@ exports.BattleItems = {
 		id: "mail",
 		name: "Mail",
 		spritenum: 403,
-		onTakeItem: false,
+		onTakeItem: function (item, source) {
+			if (this.activeMove && this.activeMove.id !== 'knockoff') return false;
```

Comment:
Whoops, this is wrong. It should say `if (!this.activeMove || this.activeMove.id !== 'knockoff')`. I'll deal with it tomorrow because I'm tired